modules/configuring-default-seccomp-profile.adoc

```diff
@@ -1,15 +1,106 @@
+// Module included in the following assemblies:
+//
+// * security/seccomp-profiles.adoc
 
-.Example
+:_content-type: PROCEDURE
 
-[source, yaml]
+[id="verifying-default-seccomp-profile_{context}"]
+= Verifying the default seccomp profile applied to a pod
+
+{product-title} ships with a default seccomp profile that is referenced as `runtime/default`. In {product-version} newly created pods have the Security Context Constraint (SCC) set to `restricted-v2` and the default seccomp profile applies to the pod. 
+
+.Procedure
+
+. You can verify the Security Context Constraint (SCC) and the default seccomp profile set on a pod by running the following commands: 
+
+.. Verify what pods are running in the namespace: 
++
+[source, terminal]
+----
+$ oc get pods -n <namespace> 
+---- 
++
+For example to verify what pods are running in the `workshop` namespace run the following: 
++
+[source, terminal]
+----
+$ oc get pods -n workshop 
+---- 
++
+.Example output 
++
+[source, terminal]
+----
+NAME                READY   STATUS      RESTARTS   AGE
+parksmap-1-4xkwf    1/1     Running     0          2m17s
+parksmap-1-deploy   0/1     Completed   0          2m22s
+---- 
++
+.. Inspect the pods: 
++
+[source, terminal]
+----
+$ oc get pod parksmap-1-4xkwf -n workshop -o yaml
+----
++
+.Example output 
++
+[source, terminal]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/network-status: |-
+      [{
+          "name": "openshift-sdn",
+          "interface": "eth0",
+          "ips": [
+              "10.131.0.18"
+          ],
+          "default": true,
+          "dns": {}
+      }]
+    k8s.v1.cni.cncf.io/networks-status: |-
+      [{
+          "name": "openshift-sdn",
+          "interface": "eth0",
+          "ips": [
+              "10.131.0.18"
+          ],
+          "default": true,
+          "dns": {}
+      }]
+    openshift.io/deployment-config.latest-version: "1"
+    openshift.io/deployment-config.name: parksmap
+    openshift.io/deployment.name: parksmap-1
+    openshift.io/generated-by: OpenShiftWebConsole
+    openshift.io/scc: restricted-v2 <1>
+    seccomp.security.alpha.kubernetes.io/pod: runtime/default <2>
+---- 
+<1> The `restricted-v2` SCC is the SCC added by default, if your workload does not have access to a different SCC. 
+<2> Newly created pods in {product-version} will have the seccomp profile configured to `runtime/default` as mandated by the SCC. 
+
+[id="upgraded_cluster_{context}"]
+== Upgraded cluster
+
+In clusters upgraded to {product-version} all authenticated users have access to the `restricted` and `restricted-v2` SCC. 
+
+If a workload is running for example on a {product-title} v4.10 cluster that was admitted by the SCC `restricted` when the cluster is upgraded that workload may get admitted by `restricted-v2` if the workload is able to run with `restricted-v2`. This is because `restricted-v2` is the most restrictive SCC between `restricted` and `restricted-v2`.
```

Comment:
The first sentence has 43 words, which is waaaaay to many. Thirty-ish is pushing it. Please rewrite this, breaking it up into two or three sentences, if possible.

Re: second sentence, if there are only two options, you can't use "most" or "least." In this case, you'd want "more restrictive."

https://www.onestopenglish.com/support-for-teaching-grammar/comparative-and-superlative-adjectives-article/144846.article